src/patch-accessors.js

```diff
@@ -386,6 +386,13 @@ let InsideAccessors = {
       children.item = function(index) {
         return children[index];
       }
+      for (const child of children) {
```

Comment:
I think we want to make a few more changes here so that this work is only done if we're not using a native method.
1. we can change the `nativeTree` implementations to just return the native result (and change where the usage site expects an array).
1. Make a more proper simple polyfill for [HTMLCollection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) and make one only if needed.

Reply:
Done